tests/online_ddl/run.sh

```diff
@@ -27,8 +27,11 @@ function real_run() {
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
+  export GO_FAILPOINTS="github.com/pingcap/dm/syncer/ExitAfterSaveOnlineDDL=return()"
```

Comment:
add this failpoint for ?

Reply:
I want to test there's no bug loading checkpoint of online DDL, so after save an online DDL checkpoint, panic and restart the worker.